docs/src/main/sphinx/connector/hive.rst

```diff
@@ -649,381 +432,412 @@ Property Name                              Description
-                                                     Example: ``https://sts.us-gov-east-1.amazonaws.com``
+Accessing tables with Athena partition projection metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``hive.metastore.glue.pin-client-to-current-region`` Pin Glue requests to the same region as the EC2 instance
-                                                     where Trino is running, defaults to ``false``.
+`Partition projection <https://docs.aws.amazon.com/athena/latest/ug/partition-projection.html>`_
+is a feature of AWS Athena often used to speed up query processing with highly
+partitioned tables.
 
-``hive.metastore.glue.max-connections``              Max number of concurrent connections to Glue,
-                                                     defaults to ``30``.
+Trino supports partition projection table properties stored in the metastore,
+and it reimplements this functionality. Currently, there is a limitation in
+comparison to AWS Athena for date projection, as it only supports intervals of
+``DAYS``, ``HOURS``, ``MINUTES``, and ``SECONDS``.
 
-``hive.metastore.glue.max-error-retries``            Maximum number of error retries for the Glue client,
-                                                     defaults to ``10``.
-
-``hive.metastore.glue.default-warehouse-dir``        Default warehouse directory for schemas created without an
-                                                     explicit ``location`` property.
-
-``hive.metastore.glue.aws-credentials-provider``     Fully qualified name of the Java class to use for obtaining
-                                                     AWS credentials. Can be used to supply a custom credentials
-                                                     provider.
+If there are any compatibility issues blocking access to a requested table when
+you have partition projection enabled, you can set the
+``partition_projection_ignore`` table property to ``true`` for a table to bypass
+any errors.
 
-``hive.metastore.glue.aws-access-key``               AWS access key to use to connect to the Glue Catalog. If
-                                                     specified along with ``hive.metastore.glue.aws-secret-key``,
-                                                     this parameter takes precedence over
-                                                     ``hive.metastore.glue.iam-role``.
+Refer to :ref:`hive_table_properties` and :ref:`hive_column_properties` for
+configuration of partition projection.
 
-``hive.metastore.glue.aws-secret-key``               AWS secret key to use to connect to the Glue Catalog. If
-                                                     specified along with ``hive.metastore.glue.aws-access-key``,
-                                                     this parameter takes precedence over
-                                                     ``hive.metastore.glue.iam-role``.
+Metastore configuration for Avro
```

Comment:
Personally I'd put this right under the `Thrift metastore configuration properties` section since it doesn't apply to Glue setups

Reply:
Ultimately though should this be on a separate Metastores page? That's what the Metastore section is for. If so, I'll move it in prep for that lift and shift to the new page.

Reply:
If you are going to put Thrift and Glue on their own pages, this `Metastore configuration for Avro` section should go in the Thrift page and not the Glue page. 

Right now, the section reads:
- Thrift
- Glue
- Partition Projection (applicable to both Thrift and Glue)
- Avro configuration (applicable only to Thrift)

Personally, if I was reading this and was using Thrift, I'd stop after seeing the Glue section and probably would not scroll down past it. Does that make sense?

Reply:
Reached out on Slack. We'll leave as-is for now and revisit when working on new Metastores page.

Reply:
@alexjo2144 - There will be one metastore page, with different sections for each supported metastore.

Reply:
However, new pages are outside the scope of this reorganization. The PR "merely" stages for that.